near-plugins/src/upgradable.rs

```diff
@@ -13,10 +13,21 @@
 //! ## Permissions
 //!
 //! Only an authorized account is allowed to call [`Upgradable::up_stage_code`] and
-//! [`Upgradable::up_deploy_code`]. There may be several reasons to protect `deploy_code`. For
-//! example if an upgrade requires migration or initialization. In that case, it is recommended to
```

Comment:
Reasoning for removing "initialization" from this paragraph: All `Upgradable` methods have parameter `self`, so they require the contract to be already initialized.